### PR TITLE
Add a simple progress indicator to `wp media regenerate`

### PR DESCRIPTION
**Pull request by nyordanov**
Adds a `$current/$count` prefix to `wp media regenerate`'s progress logs.

### CODE REVIEW

File: php/commands/media.php [hunk 1]
```diff
@@ -299,16 +299,16 @@ private function _process_regeneration( $id, $skip_delete = false, $only_missing
 			}
 
 			if ( empty( $metadata ) ) {
-				WP_CLI::warning( "Couldn't regenerate thumbnails for $att_desc." );
+				WP_CLI::warning( "$progress Couldn't regenerate thumbnails for $att_desc." );
```
Comment (danielbachhuber) [inline php/commands/media.php]: Does `$progress` ever remain empty?

**Comment by danielbachhuber** [comment]
@nyordanov Looks good, just one question.

File: php/commands/media.php (hunk 1 above)
Comment (danielbachhuber) [inline php/commands/media.php]: ... because, if it does, then we'd end up with an empty space before "Couldn't"

Comment (nyordanov) [inline php/commands/media.php]: I removed the default argument value in 01bebd9, so I should never be empty now.

**Pull request merged**